(maint) Work correctly also if PATH contains spaces

fedepell opened the pull request:
Not maybe a very common case in general, but for example easy to spot when in WSL.

puppetlabs-jenkins commented:
Can one of the admins verify this patch?

puppetcla commented:
CLA signed by all contributors.

GabrielNagy reviewed (commented):
Hi @fedepell,

Thanks for the contribution 🎉, I had some comments:
1. Travis is failing due to the commit check (you should also prefix your commit title with `(maint)`)
2. Lines 6 and 10 still run `export` with unquoted strings which will also get weird if `PATH` or `MANPATH` contain spaces. We should also wrap them in quotes.

Let me know if you have any questions.

fedepell commented:
Thanks for the feedback!
1. Ok fixed that! Sorry I'm very new to puppetlabs repositories ;)
2. Ok! I think that should have worked anyway (at least from the tests I did) while the one in _if_ really gives a bash syntax error. But for safety (and coherence in the script) indeed better to make them all quoted!

Hope it's better now! I squashed both so and also changed the title.

GabrielNagy commented on resources/files/puppet-agent.sh:
```diff
@@ -1,11 +1,11 @@
-elif ! echo ${PATH} | grep -q /opt/puppetlabs/bin ; then
-  export PATH=${PATH}:/opt/puppetlabs/bin
+elif ! echo "${PATH}" | grep -q /opt/puppetlabs/bin ; then
+  export PATH="${PATH}":/opt/puppetlabs/bin
```
I'd just enclose the entire value in quotes, like: `PATH="${PATH}:/opt/puppetlabs/bin"`, same for `MANPATH`. Works the same but in my opinion it reads better this way.

fedepell replied:
Indeed looks better as you proposed! Fixed!
